noaa_test.go

```diff
@@ -87,3 +87,17 @@ func TestChicagoOffice(t *testing.T) {
 	}
 	t.Error("noaa.Office(\"LOT\") should return valid office information.")
 }
+
+func TestChicgoHourly(t *testing.T) {
```

Comment:
Typo in test name, I assume you meant Chicago.

Reply:
D'oh... at least I didn't type 'chicano'.